regulations/templates/regulations/comment.md

```diff
@@ -0,0 +1,12 @@
+{% spaceless %}
```

Comment:
I'd be in favor of adding jinja2 for this; I like it much more than django's template language.